gobblin-core/src/main/java/gobblin/source/extractor/filebased/FileBasedSource.java

```diff
@@ -172,11 +180,13 @@ protected void initLogger(SourceState state) {
     try {
       log.info("Running ls command with input " + path);
       results = this.fsHelper.ls(path);
+      for (int i = 0; i < results.size(); i++) {
+        String filePath = state.getProp(ConfigurationKeys.SOURCE_FILEBASED_DATA_DIRECTORY) + "/" + results.get(i);
+        results.set(i, filePath + ":::" + this.fsHelper.getFileMTime(filePath));
```

Comment:
Can you make `:::` a constant, and use that constant here and in the changes to `getWorkunits`